packages/reports/tests/acceptance/report-visualizations-test.js

```diff
@@ -69,18 +69,120 @@ test('Table column sort', function(assert) {
 
   andThen(() => {
     assert.ok(find('.table-header-cell.dateTime:first .navi-table-sort-icon').hasClass('navi-table-sort-icon--asc'),
       'Datetime is sorted asc after first sort click');
-    click('.table-header-cell.dateTime:first .navi-table-sort-icon');  
+    click('.table-header-cell.dateTime:first .navi-table-sort-icon');
   });
 
   andThen(() => {
     assert.ok(find('.table-header-cell.dateTime:first .navi-table-sort-icon').hasClass('navi-table-sort-icon--desc'),
       'Datetime is sorted desc after second sort click');
-    click('.table-header-cell.dateTime:first .navi-table-sort-icon');  
+    click('.table-header-cell.dateTime:first .navi-table-sort-icon');
+  });
+
+  andThen(() => {
+    //adding a parameterized metric with a couple of parameters and run the report
+    click('.report-builder__metric-selector .grouped-list__item:contains(Platform Revenue) .grouped-list__item-label');
+    click('.metric-config__dropdown-container .grouped-list__item:contains(Euro) .grouped-list__item-label');
+    click('.navi-report__run-btn');
+  });
+
+  andThen(() => {
+    assert.notOk(find('.table-header-cell.metric:contains(Platform Revenue (USD)) .navi-table-sort-icon').hasClass('navi-table-sort-icon--desc'),
+      'Metric with first parameter is not sorted desc');
+    assert.notOk(find('.table-header-cell.metric:contains(Platform Revenue (USD)) .navi-table-sort-icon').hasClass('navi-table-sort-icon--asc'),
+      'Metric with first parameter is not sorted asc');
+    assert.notOk(find('.table-header-cell.metric:contains(Platform Revenue (EUR)) .navi-table-sort-icon').hasClass('navi-table-sort-icon--desc'),
+      'Metric with second parameter is not sorted desc');
+    assert.notOk(find('.table-header-cell.metric:contains(Platform Revenue (EUR)) .navi-table-sort-icon').hasClass('navi-table-sort-icon--asc'),
+      'Metric with second parameter is not sorted asc');
+    click('.table-header-cell.metric:contains(Platform Revenue (USD)) .navi-table-sort-icon');
+  });
+
+  andThen(() => {
+    assert.ok(find('.table-header-cell.metric:contains(Platform Revenue (USD)) .navi-table-sort-icon').hasClass('navi-table-sort-icon--desc'),
+      'Metric with first parameter is sorted desc after first sort click');
+    assert.notOk(find('.table-header-cell.metric:contains(Platform Revenue (USD)) .navi-table-sort-icon').hasClass('navi-table-sort-icon--asc'),
+      'Metric with first parameter is not sorted asc after first sort click');
+    assert.notOk(find('.table-header-cell.metric:contains(Platform Revenue (EUR)) .navi-table-sort-icon').hasClass('navi-table-sort-icon--desc'),
+      'Metric with second parameter is not sorted desc after first sort click');
+    assert.notOk(find('.table-header-cell.metric:contains(Platform Revenue (EUR)) .navi-table-sort-icon').hasClass('navi-table-sort-icon--asc'),
+      'Metric with second parameter is not sorted asc after first sort click');
+    click('.navi-report__copy-api-btn .get-api__btn');
```

Comment:
Can the`click` stmts can be move outside the `andThen`? They will still execute in the same order